<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Find_btn">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Add_btn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Delete_btn">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Swap_Btn">
        <property name="text">
         <string>Swap min and max vals</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Key_label">
        <property name="text">
         <string>Key (integer)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Key_ledit"/>
      </item>
      <item>
       <widget class="QLabel" name="Val_label">
        <property name="text">
         <string>Value (string)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Val_ledit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>Tree widget</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTravs">
    <property name="title">
     <string>Travs</string>
    </property>
    <addaction name="actionPre"/>
    <addaction name="actionPost"/>
    <addaction name="actionInfix"/>
   </widget>
   <addaction name="menuTravs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPre">
   <property name="text">
    <string>Prefix</string>
   </property>
  </action>
  <action name="actionPost">
   <property name="text">
    <string>Postfix</string>
   </property>
  </action>
  <action name="actionInfix">
   <property name="text">
    <string>Infix</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
